bitrise.yml

```diff
@@ -280,6 +280,34 @@ workflows:
         - tests_path_pattern: "integration_test"
         - additional_params: "-d iPhone --dart-define DD_CLIENT_TOKEN=$DD_CLIENT_TOKEN,DD_APPLICATION_ID=$DD_APPLICATION_ID"
 
+  integration_web:
```

Comment:
Note - this step is not run anywhere on CI yet.  We'll add it in when web support becomes more stable.